packages/ckeditor5-language/src/language.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module language/language
+ */
+
+import { Plugin } from 'ckeditor5/src/core';
+
+import LanguageEditing from './languageediting';
+import LanguageUI from './languageui';
+
+/**
+ * The language plugin.
+ *
+ * For more information about this feature check the {@glink api/language package page}.
+ *
+ * This is a "glue" plugin which loads the {@link module:language/languageediting~LanguageEditing language editing feature}
+ * and {@link module:language/languageui~LanguageUI language UI feature}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class Language extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ LanguageEditing, LanguageUI ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'Language';
+	}
+}
+
+/**
+ * The configuration of the language feature. Introduced by the {@link module:language/languageediting~LanguageEditing} feature.
+ *
+ * Read more in {@link module:language/language~LanguageConfig}.
+ *
+ * @member {module:language/language~LanguageConfig} module:core/editor/editorconfig~EditorConfig#language
+ */
+
+/**
+ * The configuration of the language feature.
+ * The option is used by the {@link module:language/languageediting~LanguageEditing} feature.
+ *
+ *		ClassicEditor
+ *			.create( {
+ * 				language: ... // Language feature config.
+ *			} )
+ *			.then( ... )
+ *			.catch( ... );
+ *
+ * See {@link module:core/editor/editorconfig~EditorConfig all editor options}.
+ *
+ * @interface LanguageConfig
+ */
+
+/**
+ * The available language options.
+ *
+ * The default value is:
+ *
+ *		const languageConfig = {
+ *			options: [
+ *				{ title: 'Arabic', class: 'ck-language_ar', languageCode: 'ar' },
+ *				{ title: 'French', class: 'ck-language_fr', languageCode: 'fr' },
+ *				{ title: 'Spanish', class: 'ck-language_es', languageCode: 'es' }
+ *			]
+ *		};
+ *
+ * The `title` and `class` properties will be used by the `language` dropdown to render available options.
+ *
+ * The `languageCode` property is used for the lang attribute in ISO 639 format. Language codes can be found
```

Comment:
As I mentioned in the other place – so far I think we've been supporting two-letter versions. But if CKE4 allowed using 3-letter ones, then it makes sense to have them covered this way here.